Fails with "Error: Cannot find module 'worker_threads'"

OS: Debian 10
Node: v10.21.0
Elm: 0.19.1

Trying to run elm-test-rs 0.4.1 for [`malaire/elm-uint64`](https://github.com/malaire/elm-uint64) fails with following error:
```
elm-test-rs 0.4.1
-----------------

Total time spent in the elm compiler: 0.13511102s
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'worker_threads'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/y/projects/GitHub/elm-uint64/elm-stuff/tests-0.19.1/js/node_supervisor.js:69:20)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,8 @@ All notable changes to this project will be documented in this file.
 
 #### Added
 
+- Support Node 10 with `--experimental-worker` option.
+
 #### Changed
 
 #### Removed

--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@ Attempt at a simpler alternative to node-test-runner for elm tests.
 Minimum supported version:
 
 - elm 0.19.1
-- Node 12
+- Node 10.5
 
 ## Usage
 

--- a/src/run.rs
+++ b/src/run.rs
@@ -351,10 +351,24 @@ fn main_helper(options: &Options, elm_project_root: &Path, reporter: &str) -> Ve
         ],
     );
 
+    let node_version = Command::new("node")
+        .arg("--version")
+        .output()
+        .expect("command failed to start")
+        .stdout;
+
+    // Node supports worker_threads as experimental feature since 10.5,
+    // but it is unknown whether all versions since 10.5 actually work with elm-test-rs.
+    let mut supervisor_args = Vec::new();
+    if node_version.starts_with(b"v10.") {
+        supervisor_args.push("--experimental-worker");
+    }
+    supervisor_args.push("js/node_supervisor.js");
+
     // Start the tests supervisor
     // eprintln!("Starting the supervisor ...");
     let mut supervisor = Command::new("node")
-        .arg("js/node_supervisor.js")
+        .args(supervisor_args)
         .current_dir(tests_root)
         .stdin(Stdio::piped())
         .spawn()

